cluster-autoscaler/README.md

```diff
@@ -27,10 +26,13 @@ there is a big chance that it won't work as expected.
 | 1.5.X  | 0.4.X  |
 | 1.4.X  | 0.3.X  |
 
-<sup>*</sup>Cluster Autoscaler 0.5.X is the official version shipped with k8s 1.6. We've done some basic tests using k8s 1.6 / CA 0.6 and we're not aware of any problems with this setup. However, CA internally simulates k8s scheduler and using different versions of scheduler code can lead to subtle issues.
+<sup>*</sup>Cluster Autoscaler 0.5.X is the official version shipped with k8s 1.6. We've done some basic tests using k8s 1.6 / CA 0.6 and we're not aware of any problems with this setup. However, Cluster Autoscaler internally simulates Kubernetes' scheduler and using different versions of scheduler code can lead to subtle issues.
```

Comment:
Recently we've had a few issues created for bugs in versions we no longer support (ex. 0.6). Basic predicates (PodsFitsResouces, PodToleratesNodeTaints, etc) haven't really changed in a long time. In most cases running a newer CA on older cluster is the best solution available to user. Maybe we should explicitly say what versions are not supported and that it is _probably_ ok to use newer CA if they're on old k8s version and run into a bug in CA.

Reply:
Do we know which versions will break for sure? 

Rephrased the earlier part to reflect that sometimes using a newer version makes sense, PTAL.